index.html

```diff
@@ -1288,6 +1286,128 @@ <h2>DID Subject</h2>
+        each <a>DID</a> in the list identifies the same <a>DID subject</a> as
+        the <a>DID</a> described by the <a>DID document</a> (i.e., the value of
+        the <code>id</code> property).
+        </p>
+        <p class="note">
+        WARNING: An assertion of semantic equivalence using the
+        <code>sameAs</code> property does not prove this assertion is true.
+        A malicious <a>DID controller</a> may attempt to impersonate a
+        <a>DID subject</a> by making a false <code>sameAs</code> assertion.
+        Therefore it is STRONGLY RECOMMENDED that a requesting party obtain
+        independent verification of a <code>sameAs</code> assertion. One method
+        of doing this is to resolve the <code>sameAs</code> <a>DID</a> to its
+        own <a>DID document</a> and then verify that it contains the inverse
+        <code>sameAs</code> assertion (i.e., a backpointer). See the
+        <a href="#security-considerations"></a> and
+        <a href="#privacy-considerations"></a> sections for more information.
+        </p>
+        </section>
+
+      <section>
+        <h2>representation</h2>
+        <p>
+       As described in Appendix A, a <a>DID</a> is an abstract identifier that
+       does not resolve directly to a <a>representation</a> of the resource
+       identified by the <a>DID</a> (the <a>DID subject</a>). Rather the
+       <a>DID resolution</a> process produces a <a>DID document</a> which in
+       turn describes the <a>DID subject</a>.
+        </p>
+        <p>
+        Appendix A classifies the <a>resources</a> identified by <a>DIDs</a>
+        into three broad categories. In the first two, the <a>DID subject</a>
+        subject is not an information resource but an entity that exists
+        independent of a digital <a>representation</a> on a network. In the
+        third, the <a>DID subject</a>  is an information resource that has a
+        digital <a>representation</a>. Common examples of this type of resource
+        include web pages, PDF files, data schemas, or digital files of any
+        type.
+        </p>
+        <p>
+        In this third case, a <a>DID controller</a> MAY use the
+        <code>representation</code> property to: 1) include a <a>representation</a>
+        of the <a>DID subject</a> directly within the <a>DID document</a>,
+        and/or 2) reference a <a>representation</a> of the <a>DID subject</a>
+        directly from the <a>DID document</a>. Since an information resource
+        may have multiple <a>representations</a>, these two options are not
+        mutually exclusive.
+        </p>
+
+        <dl>
+        <dt>representation</dt>
+        <dd>
+        The <code>representation</code> property MAY have three subproperties:
+        <code>type</code>, <code>uri</code>, and <code>value</code>.
+        </dd>
+        </dl>
+
+        <dl>
+        <dt>type</dt>
+        <dd>
+        The <code>type</code> property MUST be a <a>URI</a> that asserts the
```

Comment:
is this a URI? I would expect `application/json`, `application/ld+json`... prefer to see examples here.

Reply:
@OR13 Yes, that's a relative URI. I updated the PR to include your suggestion that the value SHOULD come from the IANA media type registry.

Reply:
I think @OR13 is stating that using `type` is the wrong property to use here?